spectree/utils.py

```diff
@@ -86,6 +93,11 @@ def parse_request(func: Any) -> Dict[str, Any]:
             "schema": {"$ref": f"#/components/schemas/{func.json}"}
         }
 
+        if model:
+            schema_extra = getattr(model.__config__, "schema_extra", None)
+            if schema_extra and has_examples(schema_extra):
+                content_items["application/json"]["examples"] = schema_extra
```

Comment:
This code checks if there's ANY dictionary with a key named `value` in `schema_extra`, and based on that it assumes that ALL items in `schema_extra` are examples. Which might not be necessarily the case:

```
class Data(BaseModel):
    value: bool

    class Config:
        schema_extra = {
            "examples": {
                "example1": {"value": {"key1": "value1", "key2": "value2"}},
                "example2": {"value": {"key1": "value1", "key2": "value2"}},
            },
            "properties": {"value": {"this is not an example": True}},
        }
```

I think it could be better to nest all examples into `schema_extra.examples`, check presence of `examples`, and then then take just that as examples:

if has_examples(...):
    content_items["application/json"]["examples"] = schema_extra.examples

What do you think?